packages/core/src/storage/vectorStore/MilvusVectorStore.ts

```diff
@@ -12,16 +13,18 @@ import type {
   VectorStoreQuery,
   VectorStoreQueryResult,
 } from "./types.js";
+import { nodeToMetadata } from "./utils.js";
```

Comment:
If you use `nodeToMetadata`, you also have to use `metadataDictToNode` see e.g. astradb